### PR TITLE
docs: improve changelog

### PR DESCRIPTION
**Pull request by simbleau**
Needs to be delivered ahead of #75 

See discussion there.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,7 +8,9 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 -->
 
-## Unreleased
+## [Unreleased]
+
+This release has an [MSRV][] of 1.80.
```
Comment (DJMcNab) [inline CHANGELOG.md:13]:
```suggestion
This release supports Bevy version 0.14 and has an [MSRV][] of 1.80.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -172,18 +196,43 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
...
 - fixed panic on Windows when window is minimized
 
-## 0.1.0 (2024-03-26)
+## [0.1.0] - 2024-03-26
+
+This release has an [MSRV][] of 1.77.
 
 - Initial release
+
+[Unreleased]: https://github.com/linebender/velato/compare/v0.5.1...HEAD
```
Comment (DJMcNab) [inline CHANGELOG.md:221]:
```suggestion
[Unreleased]: https://github.com/linebender/bevy_vello/compare/v0.5.1...HEAD
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,7 +8,9 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 -->
```
Comment (DJMcNab) [inline CHANGELOG.md:10]:
```suggestion
The latest published Bevy Vello release is [0.5.1](#051---2024-07-04) which was released on 2024-07-16.
You can find its changes [documented below](#051---2024-07-04).

```

File: CHANGELOG.md (hunk 1 above)
Comment (simbleau) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (simbleau) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,7 +8,12 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
...
 
-## Unreleased
+The latest published Bevy Vello release is [0.5.1](#051---2024-07-04) which was released on 2024-07-16.
```
Comment (DJMcNab) [inline CHANGELOG.md:11]: This date in the text was copied from Vello, and I believe it is not correct for this library.

```suggestion
The latest published Bevy Vello release is [0.5.1](#051---2024-07-04) which was released on 2024-07-04.
```

**Review by DJMcNab — APPROVED** [review]
Just one typo, otherwise looks good.

**Pull request merged**